pkg/device/device.go

```diff
@@ -0,0 +1,41 @@
+package device
```

Comment:
is this linux/posix only? might need to make build tags for unix and non-unix here